src/cmd/cgogen.go

```diff
@@ -84,14 +84,16 @@ var (
 	packagePath     = ""
 )
 
-var arrayTypes = map[string]string{
-	"PubKey":    "cipher",
-	"SHA256":    "cipher",
-	"Sig":       "cipher",
-	"SecKey":    "cipher",
-	"Ripemd160": "cipher",
-	"UxArray":   "coin",
-}
+//var arrayTypes = map[string]string{
```

Comment:
remove it if not needed

Reply:
Yes, because these specific types are sent by the configuration file, since in all the libc they are not equal.

Reply:
The code is commented out , there is no need to keep it . Please remove it

Reply:
ok